tools/replay_gen.py

```diff
@@ -374,8 +415,13 @@ def main():
     random.seed()
 
     while True:
-        f_name = pathlib.PurePath(args.output).joinpath('{0}{1}.json'.format(
-            args.prefix + '_' if args.prefix != '' else '', file_count))
+        if args.out_json:
+            f_name = pathlib.PurePath(args.output).joinpath('{0}{1}.json'.format(
+                args.prefix + '_' if args.prefix != '' else '', file_count))
+        else:
+            f_name = pathlib.PurePath(args.output).joinpath('{0}{1}.yaml'.format(
+                args.prefix + '_' if args.prefix != '' else '', file_count))
```

Comment:
I haven't removed all old `.fomat()` strings yet, but let's make new ones the f-style literals. Also, looks like we can reduce some duplication to:

```
prefix = f'{args.prefix}_' if args.prefix else ''
suffix = 'json' if out_json else 'yaml'
f_name = pathlib.PurePath(args.output).joinpath(f'{prefix}{file_count}.{suffix})')
```